Libraries/LibWeb/CSS/Interpolation.cpp

```diff
@@ -82,19 +92,24 @@ static RefPtr<CSSStyleValue const> interpolate_scale(DOM::Element& element, Calc
     auto const& from_transform = from.as_transformation();
     auto const& to_transform = to.as_transformation();
 
-    auto interpolated_x = interpolate_value(element, calculation_context, from_transform.values()[0], to_transform.values()[0], delta);
-    auto interpolated_y = interpolate_value(element, calculation_context, from_transform.values()[1], to_transform.values()[1], delta);
-
+    auto interpolated_x = interpolate_value(element, calculation_context, from_transform.values()[0], to_transform.values()[0], delta, allow_discrete);
+    if (!interpolated_x)
+        return interpolate_discrete(from, to, delta, allow_discrete);
+    auto interpolated_y = interpolate_value(element, calculation_context, from_transform.values()[1], to_transform.values()[1], delta, allow_discrete);
+    if (!interpolated_y)
+        return interpolate_discrete(from, to, delta, allow_discrete);
     RefPtr<CSSStyleValue const> interpolated_z;
 
     if (from_transform.values().size() == 3 || to_transform.values().size() == 3) {
         static auto one_value = NumberStyleValue::create(1);
         auto from = from_transform.values().size() == 3 ? from_transform.values()[2] : one_value;
         auto to = to_transform.values().size() == 3 ? to_transform.values()[2] : one_value;
-        interpolated_z = interpolate_value(element, calculation_context, from, to, delta);
+        interpolated_z = interpolate_value(element, calculation_context, from, to, delta, allow_discrete);
+        if (!interpolated_z)
+            return {};
```

Comment:
You don't fall back to `interpolate_discrete()` here, is that intentional?

Reply:
I think I just missed a place when I iinitially wrote this. The return value is now consistent for each interpolated value.